data/documentation.json

```diff
@@ -5,16 +5,19 @@
         "beaconId": "0xb5087bcd2a04f5653ad8a6b39fa4fb10fabbdec124b8b489b4d8d610e6392198",
         "name": "CoinGecko BTC/USD 0.1 percent deviation",
         "description": "The public CoinGecko BTC/USD price ticker",
-        "templateUrl": "https://github.com/api3dao/operations/blob/fa18bac5ceb1dbb894fb095a43674fe1d366e125/data/apis/api3/templates/coingecko btc_usd.json",
-        "chains": ["ropsten-2"]
-      },
-      {
-        "beaconId": "0xb5087bcd2a04f5653ad8a6b39fa4fb10fabbdec124b8b489b4d8d610e6392198",
-        "name": "CoinGecko BTC/USD 1 percent deviation",
-        "description": "The public CoinGecko BTC/USD price ticker",
-        "templateUrl": "https://github.com/api3dao/operations/blob/fa18bac5ceb1dbb894fb095a43674fe1d366e125/data/apis/api3/templates/coingecko btc_usd.json",
+        "templateUrl": "https://github.com/api3dao/operations/blob/d03fdcb6cb01c954252de63fb986b7d3e6d1dc1e/data/apis/api3/templates/coingecko btc_usd.json",
         "chains": ["ropsten"]
       }
     ]
+  },
+  "chains": {
+    "ropsten": {
+      "name": "ropsten",
+      "id": "3",
+      "contracts": { "DapiServer": "0xa70443CBE84C4e7BBcfD9DC097342932ab783e7f" },
+      "nativeToken": "ropEth",
+      "blockTime": 10,
+      "logoPath": "https://assets.coingecko.com/coins/images/13256/large/api3.jpg?1606751424"
```

Comment:
Same here?

Reply:
Documentation output is auto generated, so will match the input. I've just swapped it out.